tensorflow/contrib/rnn/python/ops/lstm_ops.py

```diff
@@ -613,6 +618,18 @@ def __init__(self,
         share weights, but to avoid mistakes we require reuse=True in such
         cases.  By default this is "lstm_cell", for variable-name compatibility
         with `tf.nn.rnn_cell.LSTMCell`.
+      input_keep_prob: unit Tensor or float between 0 and 1, input keep
```

Comment:
I'm starting to rethink this.  All of these additional arguments are basically just so you can essentially call tf.nn.dropout on the inputs?  Is that necessary?  Can't the user just call dropout themselves?

Reply:
Yes, I think so. However it might be not easy to implement variational_recurrent correctly or supports complex structure for users by themselves.

Reply:
Thanks, I rechecked the implementation of LSTMBlockFusedCell. Perhaps you're right, `tf.nn.dropout` might be good enough. Let's return to #13649 for discussion.